### PR TITLE
Update Release Guide to include cesium-omniverse-samples release process

### PR DESCRIPTION
**Pull request by lilleyse**
[Markdown Preview](https://github.com/CesiumGS/cesium-omniverse/blob/release-guide-updates/docs/release-guide/README.md)

@r-veenstra can you review? @weegeekps you may also want to review for awareness.

### CODE REVIEW

**Review by weegeekps — COMMENTED** [review]
This looks fine to me. However, for Windows I think we need to consider making an installer package to streamline the installation. It's generally fairly simple to do.

**Comment by weegeekps** [comment]
@r-veenstra Will you be reviewing this?

**Comment by r-veenstra** [comment]
> @r-veenstra Will you be reviewing this?

Yes apologies I didn't get a chance to look at today, but will do first thing in the AM.

File: docs/release-guide/README.md [hunk 1]
```diff
@@ -12,9 +12,28 @@ This is the process we follow when releasing a new version of Cesium for Omniver
...
+    * Copy the changelog into the description. Follow the format used in previous releases.
+    * Upload the Linux and Windows packages.
+12. Proceed to [Releasing a new version of Cesium for Omniverse Samples](#releasing-a-new-version-of-cesium-for-omniverse-samples).
 
-### Build Linux Package
+# Releasing a new version of Cesium for Omniverse Samples
+
+1. Create a new access token using the CesiumJS ion account.
+    * The name of the token should match "Cesium for Omniverse Samples vX.X.X - Delete on April 1st, 2023" where the version is the same as the Cesium for Omniverse release and the expiry date is two months later than present.
+    * The scope of the token should be "assets:read" for all assets.
```
Comment (r-veenstra) [inline docs/release-guide/README.md:25]: I'm making the assumption here that anything in **CesiumJS** account is intended for public access in demos etc?  

And **CesiumGS** is intended for private data, internal demos, tests, etc?

Comment (r-veenstra) [inline docs/release-guide/README.md:25]: Confirming all assets in the CesiumJS account are ok for public consumption?

**Pull request merged**